src/redux/features/categories/categorySlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
```

Comment:
- [ ] As  it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) for this project, " `Define an action type for checking the status.` ", please kindly define your action types, and the value of each action type should follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Some like so(`just an example`): 
```
const const CHECK_STATUS = micro-bookstore/books/CHECK_STATUS

```